manifests/mta.pp

```diff
@@ -26,9 +26,11 @@
 #   }
 #
 class postfix::mta (
-  Optional[Pattern[/^\S+(?:,\s*\S+)*$/]]                $mydestination = undef,
-  Optional[Pattern[/^(?:\S+?(?:(?:,\s+)|(?:\s+))?)*$/]] $mynetworks    = undef,
-  Optional[Pattern[/^\S+$/]]                            $relayhost     = undef,
+  Optional[Pattern[/^\S+(?:,\s*\S+)*$/]]                $mydestination      = undef,
+  Optional[Pattern[/^(?:\S+?(?:(?:,\s+)|(?:\s+))?)*$/]] $mynetworks         = undef,
+  Optional[Pattern[/^\S+$/]]                            $relayhost          = undef,
+  String[1]                                             $virtual_alias_maps = "hash:${postfix::confdir}/virtual",
+  String[1]                                             $transport_maps     = "hash:${postfix::confdir}/transport"
```

Comment:
This is fine, but it will only work if `postfix` was included before this class is called.

One approach to support that would be to default to `undef` and then use private variables in the class, using `pick()`:

```puppet
$_virtual_alias_maps = pick($virtual_alias_maps, "hash:${postfix::confdir}/virtual")
```